app/src/main/res/values/strings.xml

```diff
@@ -654,6 +654,9 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_bikeParkingCapacity_title">"How many bikes can be parked here?"</string>
     <string name="quest_bikeParkingCapacity_hint">"Note that most normal stands can be used from both sides to park each one bike."</string>
 
+    <string name="quest_air_pump_bicycle_title">"Is there a working bicycle air pump available to general public?"</string>
```

Comment:
What about `amenity = bicycle_repair_station access=private` with an air pump?

Maybe at least exclude tagged as `access~private|no`? ( though it is [really rare](http://overpass-turbo.eu/?q=LyoKVGhpcyBoYcSGYmVlbiBnxI1lcmF0ZWQgYnkgdGhlIG92xJJwxIlzLXR1cmJvIHdpemFyZC7EgsSdxJ9yaWdpbmFsIHNlxLBjaMSsxIk6CsOiwoDCnGFtxI1pdMSaPcSYaWN5Y2xlX3JlxKNpcl9zdMSUaW_EjmFuxJdhY2Nlc3M9cMS3dsSUxJ5nbG9ixLzFiMKdCiovCltvdXQ6anPFpF1bdGnFjMaBxoMyNV07Ci8vxI_ElMSdciDFmnN1bHRzCigKICDGlCBxdcSSxJrEo3J0IGZvcjogxYjFisWMbsWOeT1ixZPFlcWXxZnFm2HFncWfxaHGisWkIMWmxajFqsWsxa7FsGnFssSVxbrGoyBub2RlWyLFi8WNxY8iPSLGvMWUxZbFmMWaxZzFnsWgxaLFpCLGiMeax4vFrceeIseOx5Blx6vGksakd2F5x5nHm8a4x53Hn8ehxr7HpMeBx4PHqMeGbserx7vHrnPHsMeyxbPHtceTxZpsx6lux7vGt8a5x7DIgMejx4DHpseEyJPIiMetxavHr8efyI3ElciPKcaSxqbHjm7Grsabxp3GnwrGjcSYx5Z5xpI-xpLIsXNrZcS9cXQ7&c=BPgZHWMPGC&R) )

(there would be still problem with private ones that have pump - they would get tagged `service:bicycle:pump=no` but it should be ridiculously rare)

Reply:
The general idea of that _"...pump available to general public"_ phrase was primarily intended to exclude situations where e.g. bicycle repair shop has an pump for their own use, but their willingness to let you use it depends on how likeable a person asking is  - see https://github.com/streetcomplete/StreetComplete/issues/3053#issuecomment-1075853599. That's the best wording I've came with to indicate that `yes` answer should not depend on their goodwill. Improvements for wording are always welcome if someone has a better suggestion?

Anyway, I'm not sure that those `21` mapped cases worldwide are a problem requiring special handling; probably leaving a note explaining the situation so it can be mapped better in other editor is just fine in those cases. 
Note that for `access=customers` I'd leave the quest on - you should not be using that facility / be in that area if you're not a customer in any case (as warned in the intro and with signs on-site); and if you _are_ a customer who is using that facility, it is accessible to you to mark it, and is useful information for others  (e.g. _"sure I'll buy a drink/snack to be able to pump my tire here"_). 
But if people think it is a big enough problem, I could make another translation string to be asked for places marked `access=customer` which is less confusing in that case (e.g. _"Is there a working bicycle air pump available to all customers?"_)

Reply:
> probably leaving a note explaining the situation so it can be mapped better in other editor is just fine in those cases.

The problem is that mapper will answer correctly "no" and it will be tagged incorrectly. It is not clear that leaving note is necessary to avoid damaging data.

Maybe have separate questions for bicycle shop and repair station?

`access=customers` can be skipped I guess.

Reply:
Ah, good point about possible data damage if users don't leave note in that case. I've excluded `access~private|no` now. I don't see that separate questions for bicycle shop and bicycle repair stations would help here; after all it is quite possible that non-bicycle-only shop (e.g. Decathlon) has bicycle repair station in front (and may or may not enforce some rules on its use).

Perhaps a better text than _"Is there a working bicycle air pump available to general public?"_ would still help for customers case. I can think of _"Is there a working bicycle air pump available to everybody having access to this facility?"_, but that sounds just a bit too unwieldy to me. Anybody have nicer suggestion?

Reply:
it turns out that `access=customers` is invalid usage here (e747679d7331e5b3e463fe955d2f2091deadca0d), so currently suggested string is:
`Is a working bicycle air pump available here for everyone?`

That looks passable to me, but I'd be happy for improvements, provided that  that keeps following information:

- it is about `bicycle air pump` (and not some other pump, like oil pump for hydraulic bicycle brakes, or any other pump not usable for bicycle tires) 
- it is in working condition (important, as in some countries like my own it cannot be assumed at all :cry: )
- it is available to any member of general public (and not only to customers or depending on goodwill of owner etc.)